### PR TITLE
Update input-box docs to show copy when viewing in default theme

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-input-box-docs-update](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-input-box-docs-update)

File: src/app/docs/input-box/input-box-docs.component.html [hunk 1]
```diff
@@ -5,6 +5,13 @@
 >
   <sky-docs-demo-page-summary>
     Input boxes provide styling for prompts to enter data in forms. They are designed for SKY UX's modern theme but are compatible with default theme.
+
+    <ng-container
+      *ngIf="(themeSvc?.settingsChange | async)?.currentSettings?.theme?.name === 'default';"
+    >
+      COPY FOR DEFAULT-THEME GOES HERE
```
Comment (blackbaud-johnly) [inline src/app/docs/input-box/input-box-docs.component.html:12]:
```suggestion
      For the default theme, the input box is useful to handle details such as positioning input elements in the proper order. This requires less markup and avoids the need for tools such as Flexbox.
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-input-box-docs-update](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-input-box-docs-update)

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-input-box-docs-update](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-input-box-docs-update)

**Review by Blackbaud-PaulCrowder — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-input-box-docs-update](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-input-box-docs-update)

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-input-box-docs-update](https://host.nxt.blackbaud.com/skyux-forms?_sp.spa=pr-input-box-docs-update)

**Pull request merged**